.ci/watchdog/Watchdog.py

```diff
@@ -165,15 +165,15 @@ def _check_pr(self, pr):
             log.info('Ignoring PR#{}'.format(pr.number))
             return
 
-        log.info('Last status: {} at {}'.format(last_status.description, last_status.updated_at))
         # Calculate time passed since PR update (any commit, merge or comment)
         pr_time_delta = self._now_time - pr.updated_at
         # If there's no status after assumed time - check if build is waiting in queue
-        if pr_time_delta > _CI_START_THRESHOLD and not last_status:
+        if not last_status and pr_time_delta > _CI_START_THRESHOLD:
```

Comment:
What should happen if `not last_status`?
If nothing, you can just add

    if not last_status:
        return

This will simplify the condition beneath and make the code more explicit.